guides/07_FAQs.md

```diff
@@ -95,7 +95,27 @@ In Firefox, if access to camera or microphone has been granted to the site — e
 If the page itself is loaded via a different network interface than the one that is intended to be used by the Amazon Chime SDK to connect to Amazon Chime media resources, _e.g._, in a split-tunneling VPN where browser traffic uses the VPN interface but Amazon Chime video and audio does not, then ICE gathering will use the wrong interface, which can result in sub-optimal network routing or an inability to use audio or video functionality.
 
 Customers and end users must ensure that either (a) end users do not use SDK applications in these kinds of split-tunneling scenarios, or (b) the SDK application always requests microphone permissions prior to beginning ICE.
- 
+
+## Known Build Issues
+
+### Why is my build with rollup failing after upgrading to Amazon Chime JS SDK 3.8.0?
+
+In the Amazon Chime JS SDK 3.8.0 release, a bug fix for messaging was included and this bug fix added Chime SDK Messaging client form AWS JS SDK. The commit can be found here: [MessagingSession reconnects with refreshed endpoint and credentials if needed](https://github.com/aws/amazon-chime-sdk-js/commit/bce872c353edbb50908c5a0298f8113f1e8dcc82#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519).
```

Comment:
Typo. Chime SDK Messaging client form AWS JS SDK - > Chime SDK Messaging client from AWS JS SDK

Reply:
Used @devalevenkatesh's comment which does not have this typo.